high_card.rb

```diff
@@ -1,5 +1,71 @@
-# Use these two arrays to generate a deck of cards.
-ranks = ["A", 2, 3, 4, 5, 6, 7, 8, 9, 10, "J", "Q", "K" ]
-suits = [ "hearts", "spades", "clubs", "diamonds" ]
+require "pry"
 
-players = []
+
+def build_deck
+  deck = []
+  ranks = [2, 3, 4, 5, 6, 7, 8, 9, 10, "J", "Q", "K", "A"]
+  suits = [ "hearts", "spades", "clubs", "diamonds" ]
+  ranks.size.times do |i|
+    suits.each do |suit|
+    deck << [ ranks[i], suit, i+1 ]
+      # deck(ranks.with_index.to_a).map do |suit, (rank, index)|
+    end
+ end
+ return deck.shuffle
+end
+
+def get_players
+  players = []
+  while true do
+    puts "Enter Player's name or play to start: "
+    player = gets.chomp
+    if player == "play"
+      break
+    end
+    players << player
+  end
+return players
```

Comment:
I like how you are being explicit here with your returns, just know that the last line of each method is implicitly returned in Ruby